### ISSUE TITLE
ActivityModification picks up a site already modified

### ISSUE THREAD

**Issue by johnbachman** [sorgerlab/indra#43]
Here's a statement I came across from Biopax:

```
ActivityModification(MAP2K1(mods: ['PhosphorylationSerine'], mod_sites: [298]),
['PhosphorylationSerine', 'PhosphorylationSerine', 'PhosphorylationSerine'], ['218', '298', '222'],
increases, Activity)
```

MAP2K1 is listed as starting out as being phosphorylated at serine 298, but the activity modification includes that site again as being activating. Presumably this results from an error in excluding serine 298 from the modification list when examining the BioPax reaction.

**Comment by bgyori** [comment]
Did you get this from the ras pathway pathsbetween query? I think this might have been fixed as of 8db27ac8cac0d91b3ee80d291d256cd5ec02e881.

**Comment by bgyori** [comment]
This is actually interesting. The reaction this comes from says:
MAP2K1 (S298) -> MAP2K1(S298, S218, S22, active)

What you got interpreted S298 as a precondition on the event, which is not necessarily incorrect. Adding all 3 modifications as conditions of activity is also not incorrect.

An alternative interpretation could be to instead produce:

> ActivityModification(MAP2K1(),
> ['PhosphorylationSerine', 'PhosphorylationSerine'], ['218', '222'],
> increases, Activity)

which would make sense because these phosphorylations are the ones that "add" the activity. 

What do you think?

**Comment by johnbachman** [comment]
So the latter suggestion is what I originally had in mind--keep 298 as a precondition, but list 218 and 222 as the additions that activate.

From the reaction, it's clear that S298 is not sufficient for activation, but it may be necessary. If we interpret the precondition in the ActivityModification that way, that we won't be losing any information.

**Comment by bgyori** [comment]
Fixed as of 89d9fb7801236e719988526e4c3faa9ea55f0edb
